### DISCUSSION TITLE
All filenames are being logged, with VALITATE_ALL_CODEBASE=false

### DISCUSSION THREAD

**Discussion by Ziggiyzoo** [super-linter/super-linter#4871] [Q&A]
Hi,

My `.github/super-linter.env`:

VALIDATE_ALL_CODEBASE=false
IGNORE_GENERATED_FILES=true
IGNORE_GITIGNORED_FILES=true
USE_FIND_ALGORITHM=true

I was expecting this to skip all but the committed files. However, when the linter runs it logs the filenames of all files. And with the size of the repo it hasn't even reached the changed file yet after 15 mins and a tea break!

I am running locally using:

`docker run -e RUN_LOCAL=true --env-file '.github/super-linter.env' -v $PWD/workspace/er_Build_ScheduleService_$env.GIT_BRANCH:/tmp/lint github/super-linter/super-linter:latest"`

 Is this just how the linter works when run locally, or is there something I am doing wrong?

Edit: That was just it collecting a list of files.. And now it's running the linter on all files, rather than just the changed ones.

**Comment by zvailanu98** [comment]
https://github.com/super-linter/super-linter/discussions/4871#discussion-5861714

**Comment by zvailanu98** [comment]
🙏
Q&A
